static/styles/jsdoc-default.css

```diff
@@ -493,103 +650,146 @@ dl.param-type {
   color: #454545
 }
 
+
+/* tag source style */
+
+.tag-source {
+  border-bottom: 1px solid rgba(28, 160, 224, 0.35);
+}
+
+.tag-source:first-child {
+  border-bottom: 1px solid rgba(28, 160, 224, 1);
+}
+
+
 /* navicon button */
+
 .navicon-button {
-  display: none;
+  /*display: none;*/
```

Comment:
I'd prefer we just remove instead of commenting out, but I'm not passionate about it